src/metabase/query_processor.clj

```diff
@@ -40,14 +40,13 @@
 ;;; +----------------------------------------------------------------------------------------------------+
 
 
-(defn structured-query?
-  "Predicate function which returns `true` if the given query represents a structured style query, `false` otherwise."
+(defn mbql-query?
```

Comment:
This is misleading.  I consider the entire JSON based language that we have to MBQL, including both `:query` and `:native` types plus anything we may decide to add down the line.